src/components/content-tab-assistant.tsx

```diff
@@ -112,8 +112,9 @@ const UnauthenticatedView = ( { onAuthenticate }: { onAuthenticate: () => void }
 
 export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps ) {
 	const currentSiteChatContext = useChatContext();
+	const { isAuthenticated, authenticate, user } = useAuth();
 	const { messages, addMessage, clearMessages, updateMessage, chatId } = useAssistant(
-		selectedSite.name
+		user?.id ? `${ user.id }-${ selectedSite.name }` : selectedSite.name
 	);
 	const { userCanSendMessage } = usePromptUsage();
 	const { fetchAssistant, isLoading: isAssistantThinking } = useAssistantApi( selectedSite.name );
```

Comment:
Using the same logic as above, can we use selectedSite.id ?

```suggestion
	const { fetchAssistant, isLoading: isAssistantThinking } = useAssistantApi( selectedSite.id );
```

Reply:
Similar to [my previous comment](https://github.com/Automattic/studio/pull/274/files#r1647586324), I'm updating this to use the ID instead of name in https://github.com/Automattic/studio/pull/279.

Reply:
Sounds good to me. I'll follow on this after merging this work, as referenced in https://github.com/Automattic/studio/pull/274#discussion_r1647591037.